pom.xml

```diff
@@ -76,6 +76,7 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
```

Comment:
@aheritier so you like that the pom in the tag and deployed to the repository is not actually the one that the plugin was built with?

All I need to do is remember how to quote for `mvn release:prepare release:perform -Darguments=-Dmaven-compiler-plugin.version=x.y` and now there is no record of the actual configuration that was built... way to go for reproducible builds

It's an anti-pattern... I'm not saying it doesn't have its uses... but I generally find that unless you have a very specific use case or a need to sync two or more versions in the same pom, use of a property causes more problems than the three extra lines of xml that it removes... plus it hides the convention of the pom with a convention of the project which hinders new developers